rfcs/template.md

```diff
@@ -24,6 +24,8 @@ Depending on the kind of the proposal, the description should cover:
 - New use cases supported by the extension.
 - The expected performance benefit for a modification, supported with the data, if available.
 - The API of extensions such as class definitions and function declarations.
+- Key technical design aspects, sufficient to understand how the functionality should work
```

Comment:
Does this section sound too strong now given the relaxing of the process. For example, maybe "Depending on the kind of proposal, the description may cover the following. And these sections may be iteratively clarified as the proposal matures." Or something like this,

Reply:
Updated in line with your suggestion.